### DISCUSSION TITLE
Exact Steps to use the aio-onlyoffice

### DISCUSSION THREAD

**Discussion by JSChasle** [nextcloud/all-in-one#2461] [Questions] [answered]
Page https://hub.docker.com/r/nextcloud/aio-onlyoffice says to use the https://github.com/nextcloud/all-in-one but can't seem to make it work if I only do a docker run nextcloud/aio-onlyoffice (after a pull).

Installed nextcloud/all-in-one flawlessy behind a reverse proxy, just missing the way to add this container to the AIO.

Could instruction be added to the readme page?

**Comment by szaimen** [comment, marked as answer]
Hi, you may have a look at `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json`.

**Reply by Koruudo** [reply]
This JSON is for sure what you need to add the container, but the OP's request was still not answered properly, which was:
"Could instruction be added to the readme page?"

I think it'd help new users if this was added to the main GitHub readme so they can easily see how to enable OnlyOffice without digging through closed issues like this. :)

**Reply by ttytm** [reply]
This is not a proper answer to the question. Also, the one who answered this should not mark it as answer.

**Comment by Koruudo** [comment]
I'm going to append steps to the answer:

1. Edit `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json` with nano or vim
2. Locate "isOnlyofficeEnabled": 0, and change the 0 to a 1
3. Open Nextcloud-AIO console - you'll see Onlyoffice there now as a container. Stop containers, then start them back up.

P.S. Not sure why OnlyOffice isn't an "optional addon" in the original setup process? These steps should probably be in the AIO readme for others to easily see. :)

**Comment by Koruudo** [comment]
1. Open `/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json` in a text editor (vim or nano)
2. Change `"isOnlyofficeEnabled"` from **0** to **1**
3. Go to your Nextcloud-AIO console, stop containers, and then start/update them (OnlyOffice should appear in the containers list and install)

**Comment by Koruudo** [comment]
> This is not a proper answer to the question. The one who ansered this should not mark it as answer.

Agreed

**Comment by yuisheaven** [comment]
Actually, it looks to me like this config has been forgotten in the aio UI. 
Every container is available to be clicked at to have it automatically start, except for OnlyOffice.

**Reply by Zoey2936** [reply]
It was deprecated some time ago: https://github.com/nextcloud/all-in-one/discussions/1031
